factorial.cc

```diff
@@ -1,21 +1,20 @@
 // Deepak Aggarwal, Coding Blocks
 // deepak@codingblocks.com
-#include <iostream>
+#include <bits/stdc++.h> 
```

Comment:
- we should avoid bits/stdc++.h because it imports a lot of files not required. In a contest environment, it should be used as it avoids memorising imports.